include/engine_x64_nts.h

```diff
@@ -5,15 +5,24 @@ typedef int64_t zend_long;
 typedef uint64_t zend_ulong;
 typedef int64_t zend_off_t;
 
-typedef unsigned char zend_bool;
+typedef bool zend_bool;
```

Comment:
I would like to suggest to proceed with #52 first, as touching manually header files is really what I would like to avoid at all.